mpi-proxy-split/doc/nimrod-build-tutorial.txt

```diff
@@ -0,0 +1,273 @@
+This document outlines the necessary steps to take to install and compile
+NIMROD on Cori, CentOS 7 and CentOS Stream, as well as testing or running it
+natively or with MANA.
+
+A. INSTALLING AND COMPILING NIMROD
+
+1. Installing the NIMROD repository
+
+The NIMROD repository is only available to active collaborators. The
+instructions to gain access to the repository can be found here:
+
+  https://nimrodteam.org/code_access.html
+
+On Cori, the NIMROD repository has already been installed. However, it's likely
+that you will not have access to it unless it's been granted prior - email
+Jacob King (jking@txcorp.com) or an mp21 co-PI for access.
```

Comment:
See (URL) for information on how to get access.   We shouldn't name individuals in this general documentation (except as authors of MANA).